Formula/interactive-rebase-tool.rb

```diff
@@ -0,0 +1,38 @@
+class InteractiveRebaseTool < Formula
+  desc "Ncurses sequence editor for git interactive rebase"
+  homepage "https://gitrebasetool.mitmaro.ca/"
+  url "https://github.com/MitMaro/git-interactive-rebase-tool/archive/0.7.0.tar.gz"
+  sha256 "08e5d6dd9beacf7806abd74edfa9e7654ccb2ffc083b2fd8617d132951eee5bd"
+
+  depends_on "rust" => :build
+  depends_on "openssl"
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
+    ohai "Remember to configure git to use interactive-rebase-tool."
```

Comment:
No `ohai` please. This should already be in the `interactive-rebase-tool` documentation.